README.md

```
@@ -1,51 +1,104 @@
+pytest
+```
+
+Troubleshooting tips:
+- Install `pydub` (and `ffmpeg`) for local stereo-to-mono conversion.
+- Ensure `SURUS_API_BASE` if you use a custom proxy.
+
 
-# Run tests
+## Run tests
 pytest tests/
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)